### ISSUE TITLE
Padding size should be less than the corresponding input dimension, but got: padding (2, 2) at dimension 3 of input [16, 192, 2, 2]

### ISSUE THREAD

**Issue by islandLZ** [IDKiro/gUNet#11]
When I training, I meet a trouble:

at models/gunet.py : out=self.Wv(X) * self.Wg(X) 

when X'size is torch.Size([16, 192, 2, 2])， dim=192，kernel_size=5, 
self.Wg is wrong.

Could you have some method to solve this trouble?

self.Wv = nn.Sequential(
			nn.Conv2d(dim, dim, 1),   
			nn.Conv2d(dim, dim, kernel_size=kernel_size, padding=kernel_size//2, groups=dim, padding_mode='reflect')			
		)

**Comment by IDKiro** [comment]
This seems to be because you set the patch_size too small.

**Comment by islandLZ** [comment]
I set the batch is 1, so what is the block size limit?

**Comment by IDKiro** [comment]
X'size is [16, 192, 2, 2].
This means that you have set the batch_size to at least 16.

**Comment by IDKiro** [comment]
Aren't you mistaking patch_size and batch_size?

**Comment by islandLZ** [comment]
I set them all to the same.

**Comment by IDKiro** [comment]
patch_size is the size of the cropped training image, please set it to 256.
batch_size is the size of mini-batch, please set it according to your GPU memory.
It seems that you are a beginner and you might be better off looking for some simpler repositories to learn from.
